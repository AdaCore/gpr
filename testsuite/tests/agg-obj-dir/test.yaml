description:
    aggregated project attributes checks
driver: build_and_run
project_file: test
main: main
