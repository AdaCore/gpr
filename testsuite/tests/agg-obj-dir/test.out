loading files/aggl1.gpr
testing aggl1.gpr
A.Object_Dir:
B.Object_Dir:/files/b
loading files/aggl.gpr
b.gpr:2:23: warning: object directory "b" not found
aggl1.gpr:4:25: warning: library directory "lib" not found
aggl.gpr: error: aggregate library ALI directory cannot be shared with object directory of aggregated project "a"
testing aggl1.gpr
A.Object_Dir:
B.Object_Dir:/files/b
